Minor grammar/formatting cleanup around Tags and Charge Description (#280)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/columns/chargedescription.md

# Charge Description

> A Charge Description provides a high-level context of a row without requiring additional discovery. This column is a self-contained summary of what the charge is for and how it is priced. It typically covers a select group of corresponding details across a billing dataset or provides information not otherwise available.

Reviewer: ```suggestion
A Charge Description provides a high-level context of a row without requiring additional discovery. This column is a self-contained summary of the charge's purpose and price. It typically covers a select group of corresponding details across a billing dataset or provides information not otherwise available.
```
